### PR TITLE
Aminas brunch2

### PR DESCRIPTION
**Pull request by aminatkhamokova**
Privacy attack and privacy tutorials added.

### CODE REVIEW

File: holisticai/privacy/attacks/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from ._BlackBoxAttack import BlackBoxAttack, append_if_not_empty
```
Comment (981526092) [inline holisticai/privacy/attacks/__init__.py:1]: Please add the docstring description in the __init__.py file similar to the below:

"""
The :mod:`holisticai.privacy.metrics` module includes classification, regression, multiclass, recommender and clustering privacy metrics
"""

Comment (981526092) [inline holisticai/privacy/attacks/__init__.py:5]: Please add the docstring description in the init.py file similar to the below:

"""
The :mod:holisticai.privacy.metrics module includes classification, regression, multiclass, recommender and clustering privacy metrics
"""

**Review by 981526092 — CHANGES_REQUESTED** [review]

File: holisticai/privacy/attacks/__init__.py (hunk 1 above)
Comment (aminatkhamokova) [reply holisticai/privacy/attacks/__init__.py:5]: added.

**Review by 981526092 — APPROVED** [review]
Valid

**Pull request merged**